### ISSUE TITLE
Exported data cannot be imported

### ISSUE THREAD

**Issue by maximedenes** [icubam/icubam#298]
Repro steps:
```
python scripts/populate_db_fake.py --mode=dev
python scripts/csv_export.py --mode=dev --output export.csv
python scripts/csv_import.py --mode=dev --icus_csv export.csv --force_update
```

I get
```
Traceback (most recent call last):
  File "scripts/csv_import.py", line 44, in <module>
    app.run(main)
  File "/home/mdenes/dev/icubam/venv/lib/python3.8/site-packages/absl/app.py", line 299, in run
    _run_main(main, args)
  File "/home/mdenes/dev/icubam/venv/lib/python3.8/site-packages/absl/app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "scripts/csv_import.py", line 35, in main
    csv.sync_icus_from_csv(icus_f, FLAGS.force_update)
  File "/home/mdenes/dev/icubam/icubam/db/synchronizer.py", line 156, in sync_icus_from_csv
    self.sync_icus(icus_df, force_update)
  File "/home/mdenes/dev/icubam/icubam/db/synchronizer.py", line 69, in sync_icus
    manager = self._managers.get(db_icu.icu_id, self._default_admin)[0]
TypeError: 'int' object is not subscriptable
```

Tested on master's 917559ffb2fc9230aec903d6c5e93fd482f071f5

**Comment by dulacarnold** [comment]
@maximedenes can we close this?

**Comment by maximedenes** [comment]
Let me try it again, but I guess it is indeed fixed.

**Comment by maximedenes** [comment]
Yes, it works fine now, probably fixed by #304. Ideally, we should test it, but closing meanwhile. Thanks!
